channels/ae.m3u

```diff
@@ -13,16 +13,12 @@ https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m
 https://admdn4.cdn.mangomolo.com/adsports4/smil:adsports4.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="Abu Dhabi TV ARB" tvg-name="Abu Dhabi 1 ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oExmYqk.jpg" group-title="General",Abu Dhabi TV
 https://admdn2.cdn.mangomolo.com/adtv/smil:adtv.stream.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6WiXPd9.png" group-title="",Al Aan TV
+#EXTINF:-1 tvg-id="Al Aan TV ARB" tvg-name="Al Aan TV" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6WiXPd9.png" group-title="",Al Aan TV
```

Comment:
tvg-id and tvg-name should both be `Al Aan TV`

Reply:
Noted.